### PR TITLE
Portfolio: Modal section

### PR DESCRIPTION
**Pull request by Prisca-tech**
When the user clicks the button to check project details, the popup with details about the project appears.
When the user clicks the close (X) button, the popup disappears.
Implemented popups for both mobile and desktop screen sizes
Displayed HTML markup in javaScript using objects

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -320,6 +344,88 @@ textarea:focus {
...
+  align-items: center;
+  white-space: nowrap;
+}
+
+.project-description {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: left;
+  padding: 16px;
```
Comment (Mre55) [inline style.css:420]: Let's remove this `padding` that way the paragraph will start at the very left and match the Figma design provided.

File: projects.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            <ul class="flex-list">
+             ${card.Links.map((technology) => `<li>${technology}</li>`).join('')}
+            </ul>
+          </div>
+        </div>
+        <p class="project-description">${card.description}</p>
+        <div class="project-buttons">
+            <a  class="button project-button" href="${card.liveLink}">See Live
+              <img src="./Image/live-icon.svg" alt="">
+            </a>
```
Comment (Mre55) [inline projects.js:117]: Please add the proper icon for the `See Live` button to match the Figma design.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

File: style.css [hunk 3]
```diff
@@ -320,6 +344,88 @@ textarea:focus {
...
+  align-items: center;
+  text-align: center;
+  margin: 16px 0 29px 0;
+  white-space: nowrap;
+}
+
+.project-buttons {
+  display: flex;
+  gap: 12px;
+  justify-content: center;
```
Comment (Mre55) [inline style.css:410]: You can remove `justify-content: center` to align the buttons to the left. To match the design.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @Prisca-tech,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 

## Required Changes ♻️🛠️ 

- [ ] Kindly, make sure to use [HTML & CSS3 & JavaScript linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js#html--css3--javascript-course).

_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @Prisca-tech  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

### Keep Rocking!👏👏👏
<hr/>

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**